### PR TITLE
Documentation: New activity panel, emoji reactions and auto polling

### PR DESCRIPTION
**Pull request by as-op**
This includes docs on:

- Timeline view
- Fixed comment box
- Emoji reactions
- Automatic updates

# Ticket
https://community.openproject.org/projects/openproject/work_packages/58825/activity

# Merge checklist

- [x] Finish first draft
- [x] Add images
- [x] Review and hand over to marketing
- [ ] Make sure meta data is correct and up to date
- [ ] Merge

### CODE REVIEW

File: docs/user-guide/activity/README.md [hunk 1]
```diff
@@ -2,11 +2,84 @@
...
   title: Activity
   priority: 890
-description: Find out about the Activity within a project
-keywords: activity
+description: Get an overview of changes and comments within a single work package or the latest activity within a project. The activity tab in work packages also lets you mention and notify other users.
+keywords: activity, comment, mention, emoji, reaction
 ---
 
-# Activity
+# Project and work activity activity
```
Comment (MayaBerd) [inline docs/user-guide/activity/README.md]:
```suggestion
# Project and work package activity
```

File: docs/user-guide/activity/README.md [hunk 2]
```diff
@@ -2,11 +2,84 @@
...
+
+### Comments
+
+Comments allow you to have a conversation about the present work package with other project members, or comment changes you have made.
+
+![A single comment](Activity-comment.png)
+
+To add a new comment to a work package, click on the comment box at the bottom of the Activity tab. This will expand it and give you formatting options. Click on the post icon or press Command + Enter on your keyboard to publish your comment.
+
+![The comment box expands when you write your coment](Activity-commentBox.png)
```
Comment (MayaBerd) [inline docs/user-guide/activity/README.md]:
```suggestion
![The comment box expands when you write your comment](Activity-commentBox.png)
```

File: docs/user-guide/activity/README.md [hunk 3]
```diff
@@ -2,11 +2,84 @@
...
+
+The more icon (⋯) next to a comment gives you a number of additional functions.
+
+![More menu on a comment](Activity-more.png)
+
+You can:
+
+- Copy a link to the comment on your clipboard if want to paste a link to it elsewhere
```
Comment (MayaBerd) [inline docs/user-guide/activity/README.md]:
```suggestion
- Copy a link to the comment on your clipboard if you want to paste a link to it elsewhere  
```

**Comment by as-op** [comment]
Hey @psatyal @MayaBerd Please ping me before merging. I will remove all spaces from the image filenames as they are not in line with out docs style guide https://www.openproject.org/docs/development/contribution-documentation/documentation-style-guide/#directory-and-file-names

**Comment by as-op** [comment]
Please disregard my (deleted) comment. I've looked at a previous commit :D

**Comment by MayaBerd** [comment]
Thanks @psatyal! I've only changed a few typos.

File: docs/user-guide/activity/README.md [hunk 4]
```diff
@@ -2,11 +2,84 @@
...
+### Emoji reactions
+
+Starting with version 15.0, you can respond to comments with emoji reactions to quickly communicate basic messages without having to add a comment.
+
+To do so, click on the emoji icon next to each comment and pick from one of eight possible emojis.
+
+![Click the emoji icon to see a list of available emojis](Activity-addEmoji.png)
+
+You can add multiple emojis, or simply click on an emoji that was already used by someone else to add to it.
+
```
Comment (MayaBerd) [inline docs/user-guide/activity/README.md:68]:
```suggestion
> [!TIP]
> Please note that emoji reactions will not trigger notifications. If you need your colleague to be notified about your reaction, leave a regular comment.

```

**Pull request merged**